modules/apps/object/object-web/src/main/resources/META-INF/resources/js/tests/ListTypeDefinition/ListTypeDefinition.spec.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+
+import {toCamelCase} from '../../utils/string';
+
+it('Assert string format depending on context', () => {
```

Comment:
Hey @rijema, there are many things we can improve in this test. I write all of them in this one comment here.

1. toCamelCase is not used only by picklists. This is simply a string util. Please, change the location/filename to `tests/utils/string.spec.ts`, instead of `tests/ListTypeDefinition/ListTypeDefinition.spec.tsx`
2. Use describe like the example below to ease the understanding of parameters;
3. Write one case per assertion;
4. test the actual camelCase transformation

Example:
```
describe('toCamelCase(str, removeSpecialCharacters, keepFirstLetterCase)', () => {
	it('returns first letter in uppercase', () => {
		expect(toCamelCase('FirstLetterUpperCase', false, true)).toBe('FirstLetterUpperCase');
	});

	it('returns first letter in lowercase, () => {
		expect(toCamelCase('firstLetterLowerCase', false, true)).toBe('firstLetterLowerCase');
	});

	it('removes special characters, () => {
		expect(toCamelCase('specialCharacter$', false, true)).toBe('specialCharacter');
	});

	it('makes UPPER CASE string camelCase, () => {
		expect(toCamelCase('UPPER CASE', false, false)).toBe('upperCase');
	});

	it('makes lower case string camelCase, () => {
		expect(toCamelCase('lower case', false, false)).toBe('lowerCase');
	});
});
```
Plase, check if above assertions are correct and work.

Reply:
Hi, sir! I changed two of them, as the toCamelCase method doesn't change the structure of the remaining chars, but the first ones in each word.